CenterForOpenScience/osf.io
# [WIP][ENG-2715] Remove Prereg landing page and campaign info

## Purpose

Removes now unneeded prereg campaign code. 

## Changes

- removes landing pages, campaign logic, tests, permissions for prereg campaign
- redirects `/prereg/` to the new workflow

## QA Notes

Please make verification statements inspired by your code and what your code touches.
- Verify that redirect works
- Verify

What are the areas of risk?

Any concerns/considerations/questions that development raised?

## Documentation

Tech task, no new docs.

## Side Effects

None that I know of.

## Ticket

https://openscience.atlassian.net/browse/ENG-2715
